### PR TITLE
Align readme format

### PR DESCRIPTION
**Pull request by Goltergaul**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Contributor Covenant Code of Conduct
```
Comment (nilsding) [inline CODE_OF_CONDUCT.md:1]: ❤️

File: README.md [hunk 2]
```diff
@@ -38,98 +80,48 @@ end
 
 ### Sinatra
 
-Add to config.ru
+Add the middleware to your config.ru
```
Comment (nilsding) [inline README.md]: Please wrap file names with backticks `so they show up like this`

File: README.md [hunk 3]
```diff
@@ -38,98 +80,48 @@ end
...
-As soon as the middleware is configured and your application is running, you can
-acess the health monitor status page adding 'health_monitor' to your url. It does
-not depend on the given path.
-
-    http://your_application_url_with_some_path/health_monitor
+### Service monitors
 
-This would lead to the following output:
+These can be used to monitor another service that is also using this gem. The block
+needs to return the result of the /healthmonitor endpoint of the other service:
```
Comment (nilsding) [inline README.md]: By default, this gem mounts itself to `/health_monitor`.  Please update this + the example according to that, and wrap the path in backticks

File: README.md [hunk 4]
```diff
@@ -148,3 +140,22 @@ task :environment do
   require_relative "./config/environment" if File.exists?("./config/environment")
 end
 ```
+
+## Development
+
+After checking out the repo, run `bin/setup` to install dependencies. You can also run `bin/console` for an interactive prompt that will allow you to experiment.
```
Comment (nilsding) [inline README.md]: Please wrap the lines at around 80 characters.

**Review by nilsding — CHANGES_REQUESTED** [review]
Generally nice, just some minor formatting things.

File: README.md [hunk 5]
```diff
@@ -38,98 +80,48 @@ end
...
-As soon as the middleware is configured and your application is running, you can
-acess the health monitor status page adding 'health_monitor' to your url. It does
-not depend on the given path.
-
-    http://your_application_url_with_some_path/health_monitor
+### Service monitors
 
-This would lead to the following output:
+These can be used to monitor another service that is also using this gem. The block
+needs to return the result of the /health_monitor endpoint of the other service:
```
Comment (nilsding) [inline README.md]: Backticks ;-) 

```suggestion
needs to return the result of the `/health_monitor` endpoint of the other service:
```

**Review by nilsding — APPROVED** [review]

**Comment by Goltergaul** [comment]
does github do autosquash on merge?

**Comment by nilsding** [comment]
Don't think so, but it can do squash merges.

**Pull request merged**